cirq/google/api/v2/program.proto

```diff
@@ -276,25 +276,25 @@ message MeasurementKey {
 // swap measurement keys a and b.
 message MeasurementKeyMapping {
   // Indicates that measurement key "key" should be replaced with "value".
-  message Entry {
+  message MeasurementKeyEntry {
```

Comment:
For consistency, the other two should be `QubitEntry` and `ArgEntry`.

Reply:
Oh good point with the consistency - I would call this instead `MeasurementKeyMappingEntry` then...? Too long?

Reply:
+1 for `QubitEntry`, `ArgEntry`, `MeasurementKeyEntry`.

Reply:
Echoing @maffoo: these will always appear as `*Mapping.<name>`. We only need enough prefix to dedupe these types, so I also prefer `*Entry` instead of `*MappingEntry`.

Reply:
gotcha, fixing!